src/js/login/containers/Main.jsx

```diff
@@ -3,12 +3,17 @@ import { connect } from 'react-redux';
 import appendQuery from 'append-query';
 
 import recordEvent from '../../../platform/monitoring/record-event';
-import Modal from '@department-of-veterans-affairs/jean-pants/Modal';
 import { getUserData } from '../../common/helpers/login-helpers';
+import { isUserRegisteredForBeta } from '../../personalization/beta-enrollment/actions';
```

Comment:
This function is not an action, but it's written as one. Not something you need to fix, but it was confusing when reading through the code.

Reply:
I agree. Doesn't end up dispatching anything and simply checks something in the current state. But looks like something the personalization team was working on.